tests/integration/test_hermes_adopt_existing_install.py

```
@@ -0,0 +1,240 @@
+
+
+class _AuthorityHandler(BaseHTTPRequestHandler):
+    def do_POST(self) -> None:  # noqa: N802
+        if self.path in {"/policy/authority/onboard", "/policy/authority/evaluate"}:
+            self.send_response(405)
+            self.end_headers()
+            return
+        self.send_response(404)
+        self.end_headers()
+
+    def log_message(self, format: str, *args: object) -> None:  # noqa: A003
```

Ruff (0.15.10)

[error] Function argument `format` is shadowing a Python builtin

(A002)